SIT.Manager.Avalonia/Views/MainView.axaml

```diff
@@ -25,12 +25,18 @@
 			</ui:TeachingTip.IconSource>
 		</ui:TeachingTip>
 
-		<StackPanel Name="InfoBarStackPanel" Grid.Row="0" x:FieldModifier="Public" IsVisible="True">
-			<ui:InfoBar Name="UpdateInfoBar" IsOpen="False">
-				<ui:InfoBar.ActionButton>
-					<Button Name="UpdateButton" Content="Update" Command="{Binding UpdateButtonCommand}" HorizontalAlignment="Right"/>
-				</ui:InfoBar.ActionButton>
-			</ui:InfoBar>
+		<StackPanel Name="InfoBarStackPanel" Grid.Row="0" x:FieldModifier="Public">
+			<Grid ColumnDefinitions="auto, auto, auto, *, auto" IsVisible="{Binding UpdateAvailable}">
+				<Canvas Margin="5 0 0 17" VerticalAlignment="Center" Grid.Column="0">
+					<Ellipse Fill="#1e4f8a" Width="18" Height="18"/>
+					<ui:SymbolIcon Symbol="Important" VerticalAlignment="Center" Margin="2 1 0 0"/>
+				</Canvas>
+				<TextBlock Text="An update to the SIT Manager is available click here to update:" Grid.Column="1" Margin="28 0 0 0" VerticalAlignment="Center"/>
```

Comment:
Using a panel set with `Margin="14,0"` instead of a canvas here would avoid the need for having a margin set on the TextBlock, and also then line the icon up with the menu below and that would be consistent with how the bar notifications happen as well.